composer.json

```diff
@@ -20,7 +20,7 @@
     "GPL-2.0+"
   ],
   "require": {
-    "typo3/cms-core": ">=7.6.0,<8.9.99"
+    "typo3/cms": ">=7.6.13,<8.9.99"
```

Comment:
I am absolutly fine to force 8x users to latest version. So what about `"typo3/cms-core": ">=7.6.13 || >=8.5.0 <8.9.99"`

Would like to use the `-core`, following composer.json of typo3_console